### PR TITLE
feat: integrate invite members API

### PR DESCRIPTION
**Pull request by Meldy2503**

### CODE REVIEW

File: src/types/index.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+export type CursorType = { cursor: string; isCurrent: boolean; page: number };
+
```
Comment (Christiano112) [inline src/types/index.tsx:2]: Duplicate file

**Review by Christiano112 — APPROVED** [review]

**Review by Christiano112 — CHANGES_REQUESTED** [review]
Requested changes

**Review by TunjiDev — APPROVED** [review]

File: src/app/auth/signin/page.tsx [hunk 2]
```diff
@@ -51,7 +57,11 @@ function SignIn() {
                 toast.success("Sign In Successful!" || result.data.message);
                 sessionStorage.setItem("user", JSON.stringify(result.data.data));
 
-                return router.push(`/dashboard/${userId}`);
+                if (typeof window !== "undefined" && sessionStorage.getItem("groupCode")) {
```
[the remark this answers is not in the document]
Comment (Meldy2503) [reply src/app/auth/signin/page.tsx:60]: I tried that but groupCode && ... but the build failed, it was complaining about the session storage

File: src/components/groups/group-table.tsx [hunk 3]
```diff
@@ -1,90 +1,161 @@
...
+        ...{ ...filters, size: selectedPageSize },
+    });
+
+    const memoizedData = useMemo(() => {
+        if (memberSuccess) return reformData(membersData);
+        return [];
+    }, [membersData, memberSuccess]);
+
+    console.log(membersData);
+    // console.log(memoizedData.id)
```
[the remark this answers is not in the document]
Comment (Meldy2503) [reply src/components/groups/group-table.tsx]: done

File: src/types/index.tsx (hunk 1 above)
Comment (Meldy2503) [reply src/types/index.tsx:2]: done, thanks

File: src/app/auth/signin/page.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (Christiano112) [reply src/app/auth/signin/page.tsx:60]: Ohhkay

**Review by Christiano112 — APPROVED** [review]

**Pull request merged**